README.md

```diff
@@ -1,30 +1,80 @@
 # LangageDessinVectoriel
 
-This project aims to create SVG images through the command of a user.
+This project aims to create vector graphics through the command of user, and graphics that are created could be showed in a java frame and saved as a SVG file.
 
 The subject of the subject is described [here](http://www.grall.name/teaching/softwareArchitecture/2016/project_vectorGraphics.html) (in French)
 
-## How to use the software:
+## About the software:
 
-Once the project is retrieved, you can use it to create SVG files.
-In the package src/view, you have a file "Draw.java". This is the java class you'll have to use to run the software.
+Once the project is retrieved, you can use it to create graphics. With this project, you have two ways to draw an image. Firstly, using the **GUI** to draw, which is easy to use; or you can use a **drawing language** based on Java, which is less intuitive but you will gain more control from it.
 
-### Step 1:
-In the main method, you'll first need to create a Pencil (or some Pencils) to draw.
-`Pencil p = new Pencil(4, new ColorSVG(Color.black.getRGB()));`
-Pencil takes 2 arguments: the width of the line that will be drawn, and the color of that line (which uses a ColorSVG class, which takes an int as argument (RGB int)).
+In the package src/view, you will find 3 classes needed:
 
-### Step 2:
-You'll then need some shapes to draw.
-The list of shapes are for now listed in src/model/ShapeName.
-Each type of shape has it's own attribute. So for now, you'll need to see what they are for each specific shapes.
-They are located in src/model/specific_path.
-As an example: `Circle c = new Circle(50,40,30);`
-You are encourage to add a color to your shape like that: `c.setColor(new ColorSVG(Color.blue.getRGB()));`
-As well as defining the pencil used to draw that shape, pencil you would already have created: `c.setPencil(p);`
+- `view.GUI`: This is the entrance for our GUI.
+- `view.Painting`: Here is the "drawing language" that we defined. It is a script that you could modify to tell the software what it should draw.
```

Comment:
All this lines will form only one line if you don't skip another line.

`It is a script that you could modify to tell` could you change it into `It is a script that you can modify to tell`

Or you can do the read me in another PR actually, so this PR can be merged sooner